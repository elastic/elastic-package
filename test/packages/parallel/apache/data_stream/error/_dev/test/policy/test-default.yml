data_stream:
  vars:
    paths:
      - "/var/logs/apache/error.log*"
