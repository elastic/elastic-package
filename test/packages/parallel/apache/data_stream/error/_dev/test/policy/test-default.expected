agent:
    download:
        sourceURI: https://artifacts.elastic.co/downloads/
    features: {}
    monitoring:
        enabled: false
        logs: false
        metrics: false
    protection:
        enabled: false
inputs:
    - data_stream:
        namespace: ep
      meta:
        package:
            name: apache
      name: test-default-apache
      revision: 1
      streams:
        - data_stream:
            dataset: apache.error
            type: logs
          exclude_files:
            - .gz$
          paths:
            - /var/logs/apache/error.log*
          processors:
            - add_locale: null
          tags:
            - apache-error
      type: logfile
      use_output: default
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - logs-apache.error-ep
                  privileges:
                    - auto_configure
                    - create_doc
secret_references: []
